beater/cloudbeat.go

```diff
@@ -47,15 +44,13 @@ type cloudbeat struct {
 	data        *manager.Data
 	evaluator   evaluator.Evaluator
 	transformer transformer.Transformer
+	log         *logp.Logger
 }
 
-const (
-	cycleStatusStart = "start"
-	cycleStatusEnd   = "end"
-)
-
 // New creates an instance of cloudbeat.
 func New(b *beat.Beat, cfg *common.Config) (beat.Beater, error) {
+	log := logp.NewLogger("cloudbeat")
```

Comment:
I am wondering why did you decide to use "new logger here".

I tried to explore and research logging in general and logging good practices in Go.
As far as I read, I didn't see any objection to using one global log, unless there is a specific reason.

I tried to also to find some examples in the "beats" Repo and I couldn't find any other similar example (where the beat creates a new logger this way).

Most of our cloudbeat uses the following:
`logp.L().Info("some message")`

(which refers to a global log).

Why do you think there is a need to creates a new logger here ?

Reply:
this is about following go practices, but following elastics. see [here](https://github.com/elastic/beats/blob/eeda98b82f776d4f9104f2fad3958bdf291e8f59/x-pack/osquerybeat/beater/osquerybeat.go#L74
) for example

Creating a logger in this manner will allow us to filter its log by the key in Kibana. [Here](https://github.com/elastic/beats/blob/eeda98b82f776d4f9104f2fad3958bdf291e8f59/libbeat/logp/logger.go#L227) they mention that using .L() will create an unnamed logger, as we see in the logs in Kibana.